arrows/core/derive_metadata.cxx

```diff
@@ -210,47 +237,86 @@ double
 compute_gsd( kwiver::vital::metadata_sptr const& metadata,
              size_t frame_width, size_t frame_height )
 {
-  if ( frame_width == 0 || frame_height == 0 )
+  if( frame_width < 1 || frame_height < 1 )
   {
-    VITAL_THROW( kv::invalid_value, "frame dimensions cannot be zero" );
+    VITAL_THROW( kv::invalid_value, "frame dimensions must both be positive" );
   }
 
-  kv::vector_2d sensor_fov;
+  // RP1201 method
   try
   {
-    // Attempt to acquire sensor FOV
-    sensor_fov = get_sensor_fov( metadata );
+    // Intermediate values
+    kv::rotation_d const total_rotation = get_total_rotation( metadata );
+    double yaw, pitch, roll;
+    total_rotation.get_yaw_pitch_roll( yaw, pitch, roll );
+    // Ideally slant range should be pre-calculated
+    double const slant_range = get_slant_range( metadata );
```

Comment:
This changed from `compute_slant_range` to `get_slant_range`, which seems inconsistent with the comment. In light of a later comment, it seems this change may have been inadvertent?

Reply:
The comment is wrong and the implementation is correct. The code which computes `slant_range` is unreachable in the reference implementation.